src/neo-vm/ExecutionEngine.cs

```diff
@@ -340,8 +340,7 @@ private void ExecuteInstruction(Instruction instruction)
                     }
                 case OpCode.THROW:
                     {
-                        UncaughtException = Pop();
```

Comment:
How the syscall will call this opcode, when syscall throw an exception it goes to `OnFault`

Reply:
They should call the method `Throw()`.

https://github.com/neo-project/neo-vm/blob/98cde1d9eb920033d39dec39c13bba23cb76090a/src/neo-vm/ExecutionEngine.cs#L1451